### PR TITLE
Fix/add border less rows support

### PR DESCRIPTION
**Pull request by mjmonline**

### CODE REVIEW

File: src/components/table-content-wrapper/table-body/table-body-row/table-body-row.jsx [hunk 1]
```diff
@@ -46,20 +51,20 @@ class TableBodyRow extends Component {
   };
 
   render() {
-    const { classes, rowData, columns, renderTd, expandLabel, trProps } = this.props;
+    const { classes, rowData, columns, renderTd, expandLabel, trProps, borderLessRows } = this.props;
```
Comment (JacobBlomgren) [inline src/components/table-content-wrapper/table-body/table-body-row/table-body-row.jsx]: borderless is one word, so should be `borderlessRows`

File: src/components/table-content-wrapper/table-body/table-body-row/table-body-row.jsx [hunk 2]
```diff
@@ -46,20 +51,20 @@ class TableBodyRow extends Component {
...
-        {columns.map(col => (
+        {colsWithRows.map(col => (
           <Td
             className={cn({
               [classes.firstMobileRow]: col.firstMobileRow,
             })}
             key={col.key || col.baseKey || col.padding}
             align={col.align}
-            borderBottom
+            borderBottom={collapsed || borderLessRows.indexOf(col.rowNr) === -1}
```
Comment (JacobBlomgren) [inline src/components/table-content-wrapper/table-body/table-body-row/table-body-row.jsx]: seems like `indexOf` could be substituted with [`includes`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes).

File: src/components/table-content-wrapper/table-body/table-body-row/table-body-row.jsx [hunk 3]
```diff
@@ -75,7 +80,7 @@ class TableBodyRow extends Component {
           flexOrder={spacers.shift()}
           flexBasisMobile={10}
           ellipsis={false}
-          borderBottom
+          borderBottom={collapsed || borderLessRows.indexOf(1)}
```
Comment (JacobBlomgren) [inline src/components/table-content-wrapper/table-body/table-body-row/table-body-row.jsx]: is `includes` appropriate here too?

File: src/components/table-content-wrapper/table-body/table-body-row/table-body-row.jsx [hunk 4]
```diff
@@ -87,10 +92,10 @@ class TableBodyRow extends Component {
             )}
           </button>
         </Td>
-        {spacers.map(spacerIdx => (
+        {spacers.map((spacerIdx, index) => (
```
Comment (JacobBlomgren) [inline src/components/table-content-wrapper/table-body/table-body-row/table-body-row.jsx]: If `idx` is abbreviated once it should be everywhere, consistently.

File: src/components/table-content-wrapper/table-body/table-body-row/table-body-row.jsx (hunk 1 above)
Comment (mjmonline) [reply src/components/table-content-wrapper/table-body/table-body-row/table-body-row.jsx]: fixed

File: src/components/table-content-wrapper/table-body/table-body-row/table-body-row.jsx (hunk 2 above)
Comment (mjmonline) [reply src/components/table-content-wrapper/table-body/table-body-row/table-body-row.jsx]: fixed

File: src/components/table-content-wrapper/table-body/table-body-row/table-body-row.jsx (hunk 3 above)
Comment (mjmonline) [reply src/components/table-content-wrapper/table-body/table-body-row/table-body-row.jsx]: fixed

File: src/components/table-content-wrapper/table-body/table-body-row/table-body-row.jsx (hunk 4 above)
Comment (mjmonline) [reply src/components/table-content-wrapper/table-body/table-body-row/table-body-row.jsx]: fixed

**Pull request merged**